### PR TITLE
User login and logout 

### PR DESCRIPTION
**Pull request by rmoldo**
Implemented user login and logout and configured glassfish server for those operations.

### CODE REVIEW

File: src/main/java/com/park/easyrecruit/servlet/Login.java [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/**
+ *
+ * @author M Radu
+ */
+@WebServlet(name = "Login", urlPatterns = {"/Login"})
```
Comment (dragosdinescu) [inline src/main/java/com/park/easyrecruit/servlet/Login.java:15]: lowercase login maybe?

Comment (rmoldo) [reply src/main/java/com/park/easyrecruit/servlet/Login.java:15]: I followed the convention used by Alex Dorobantiu in 10b-D.Reorganize URLs.

File: src/main/webapp/WEB-INF/pages/navbar.jsp [hunk 2]
```diff
@@ -65,7 +62,17 @@
                 <li class="nav-item">
                     <a class="nav-link" href="${contextPath}/Profile">Profile</a>
                 </li>
-            </c:if>
+            </c:if> 
+            <li class="nav-item">
+                <c:choose>
+                        <c:when test="${pageContext.request.getRemoteUser() == null}">
```
Comment (ghost) [inline src/main/webapp/WEB-INF/pages/navbar.jsp:68]: Does this check return null anytime an user is not logged in? Doesn't have any other corner cases?
I think the logic there should be to show logout when user is logged in. Any other situation, the login should be displayed. Are you sure this type of check completely covers the logic?

**Review by ghost — APPROVED** [review]

File: src/main/webapp/WEB-INF/pages/navbar.jsp (hunk 2 above)
Comment (rmoldo) [reply src/main/webapp/WEB-INF/pages/navbar.jsp:68]: java.lang.String getRemoteUser(): Returns the login of the user making this request, if the user has been authenticated, or null if the user has not been authenticated. 

https://docs.oracle.com/javaee/6/api/javax/servlet/http/HttpServletRequest.html#getUserPrincipal%28%29

**Pull request merged**